configuration/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### Unreleased
 
+- adds transaction submitters type and to replace transaction signers
```

Comment:
There's a typo here. Can this be expanded?

Reply:
Nvm about expanding, I see that these have been spread out

Reply:
Yeah fixed typo